src/extensions/mvs/tree/mvs/mvs-builder.ts

```diff
@@ -209,6 +219,20 @@ export class Component extends _Base<'component' | 'component_from_uri' | 'compo
 }
 
 
+/** MVS builder pointing to a 'volume_representation' node */
+export class VolumeRepresentation extends _Base<'volume_representation'> {
+    /** Add a 'color' node and return builder pointing back to the volume_representation node. 'color' node instructs to apply color to a visual representation. */
+    color(params: ParamsOfKind<MVSTree, 'color'>): VolumeRepresentation {
+        this.addChild('color', params);
+        return this;
+    }
+    /** Add a 'color_from_uri' node and return builder pointing back to the volume_representation node. 'color_from_uri' node instructs to apply colors to a visual representation. The colors are defined by an external annotation resource. */
+    colorFromUri(params: ParamsOfKind<MVSTree, 'color_from_uri'>): VolumeRepresentation {
+        this.addChild('color_from_uri', params);
+        return this;
+    }
+}
```

Comment:
We don't have `color_from_uri` on `volume_representation` in backend, and it doesn't even make much sense at this stage, as there are no selectors which could be applied to volumes.

Reply:
Yes, that's absolutely correct, thanks for the catch.